kalite/topic_tools/tests/utils_tests.py

```diff
@@ -0,0 +1,47 @@
+from mock import patch, MagicMock
+
+import kalite.topic_tools as mod
+from kalite.testing.base import KALiteTestCase
+
+
+DUMMY_STRING = "maize"
+TRANS_STRING = "trans"
+
+
+ugettext_dummy = MagicMock(return_value=DUMMY_STRING)
+
+
+@patch.object(mod, '_', new=ugettext_dummy)
+class SmartTranslateItemDataTests(KALiteTestCase):
```

Comment:
Perhaps good to include a test where we don't expect internationalization, like if `a in ASSESSMENT_DATA_TRANSLATABLE_FIELDS` is `False`:
```python
def test_not_translated(self):
    test_data = {a: {'content': 'this should not be translated'}}
    # assert that ugettext_dummy called zero times
```